src/main/java/com/raoulvdberge/refinedstorage/apiimpl/network/node/NetworkNodeGrid.java

```diff
@@ -491,9 +491,13 @@ public void onCreatePattern() {
             patterns.setStackInSlot(1, pattern);
         }
     }
+    
+    private boolean patternAvailable() {
```

Comment:
`patternAvailable` is a weird name for a method that checks if two pattern slots AREN'T available...

Reply:
its supposed to return the opposite, sorry! commited that change